static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -486,9 +486,22 @@ define([
                     bonzo(geoMostPopular.elem).remove();
                 });
             });
-        }
+        },
+        '88,70': isFluid250('ad-slot--top-banner-ad'),
+        '88,71': isFluid250('ad-slot--top-banner-ad')
+        '88,88': isFluid250('ad-slot--commercial-component'),
     };
 
+    function isFluid250(className) {
```

Comment:
I'm not quite sure I follow this - are these adverts actually fluid250s? If so, can we not just make `fluid250.js` add the class to its container?